# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1296367120 28800
#      Sat Jan 29 21:58:40 2011 -0800
# Node ID 0ce4a9cd6a7fa73b223564292cdad7a9176dcc48
# Parent  d5f50ea57869da10906151c6b8d0077f8d43867e
Add accumdim function to documentation.c

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-29  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/arith.txi: Add accumdim function to documentation.
+
 2011-01-29  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/set.txi: Add powerset function to documentation.
 
 2011-01-29  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/system.txi: Add S_ISSOCK function to documentation.
 
diff --git a/doc/interpreter/arith.txi b/doc/interpreter/arith.txi
--- a/doc/interpreter/arith.txi
+++ b/doc/interpreter/arith.txi
@@ -186,16 +186,18 @@ cos (pi/2)
 @DOCSTRING(cumsum)
 
 @DOCSTRING(cumprod)
 
 @DOCSTRING(sumsq)
 
 @DOCSTRING(accumarray)
 
+@DOCSTRING(accumdim)
+
 @node Utility Functions
 @section Utility Functions
 
 @DOCSTRING(ceil)
 
 @DOCSTRING(curl)
 
 @DOCSTRING(cross)
diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-01-29  Rik  <octave@nomad.inbox5.com>
+
+	* general/accumarray.m: Add seealso reference to accumdim.
+
 2010-01-29  Rik  <octave@nomad.inbox5.com>
 
 	* set/powerset.m: Improve docstring.
 
 2010-01-29  Rik  <octave@nomad.inbox5.com>
 
 	* miscellaneous/copyfile.m, miscellaneous/movefile.m,
 	miscellaneous/tempname.m: Improve docstrings
diff --git a/scripts/general/accumarray.m b/scripts/general/accumarray.m
--- a/scripts/general/accumarray.m
+++ b/scripts/general/accumarray.m
@@ -57,16 +57,17 @@
 ## number of
 ## subscripts and M is the maximum subscript (linearized in multi-dimensional
 ## case).
 ## If @var{func} is one of @code{@@sum} (default), @code{@@max}, @code{@@min}
 ## or @code{@@(x) @{x@}}, an optimized code path is used.
 ## Note that for general reduction function the interpreter overhead can play a
 ## major part and it may be more efficient to do multiple accumarray calls and
 ## compute the results in a vectorized manner.
+## @seealso{accumdim}
 ## @end deftypefn
 
 function A = accumarray (subs, vals, sz = [], func = [], fillval = [], issparse = [])
 
   if (nargin < 2 || nargin > 6)
     print_usage ();
   endif
 
